gui/src/ocpn_app.cpp

```diff
@@ -1128,13 +1146,15 @@ bool MyApp::OnInit() {
   temp_font.SetDefaultEncoding(wxFONTENCODING_SYSTEM);
 
   //      Establish Log File location
-  if (!g_Platform->InitializeLogFile()) return false;
+  if (!g_Platform->InitializeLogFile()) {
+    return false
+  };
 
 #ifdef __WXMSW__
 
-    //  Un-comment the following to establish a separate console window as a
```

Comment:
The CI uses a slightly old version of `clang-format` which causes these code comments to be incorrectly indented. It is indented as if these comments belonged inside if above `if (!g_Platform->InitializeLogFile())` statement.
A newer version of `clang-format` properly indents code. A workaround is to add curly braces such that both the older version and the newer version of `clang-format` indent code the same way.